### PR TITLE
New version and Move: VPNetwork.TorGuard version 4.8.6-build.128.1+g49783c1

### PR DESCRIPTION
**Pull request by OfficialEsco**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/41663)

### CODE REVIEW

File: manifests/t/TorGuard/TorGuard/4.8.6-build.128.1+g49783c1/TorGuard.TorGuard.installer.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
 
 PackageIdentifier: TorGuard.TorGuard
```
Comment (Trenly) [inline manifests/t/TorGuard/TorGuard/4.8.6-build.128.1+g49783c1/TorGuard.TorGuard.installer.yaml:4]:
```suggestion
PackageIdentifier: VPNetwork.TorGuard
```

File: manifests/t/TorGuard/TorGuard/4.8.6-build.128.1+g49783c1/TorGuard.TorGuard.locale.en-US.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
 
 PackageIdentifier: TorGuard.TorGuard
```
Comment (Trenly) [inline manifests/t/TorGuard/TorGuard/4.8.6-build.128.1+g49783c1/TorGuard.TorGuard.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: VPNetwork.TorGuard
```

File: manifests/t/TorGuard/TorGuard/4.8.6-build.128.1+g49783c1/TorGuard.TorGuard.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
 
 PackageIdentifier: TorGuard.TorGuard
```
Comment (Trenly) [inline manifests/t/TorGuard/TorGuard/4.8.6-build.128.1+g49783c1/TorGuard.TorGuard.yaml:4]:
```suggestion
PackageIdentifier: VPNetwork.TorGuard
```

**Review by Trenly — COMMENTED** [review]
@OfficialEsco Since there is only 1 version of this in the repo, can we maybe update the ID to be in the form Publisher.Package ?

**Review by ImJoakim — APPROVED** [review]

**Pull request merged**